provider/coredns/coredns.go

```diff
@@ -302,8 +302,8 @@ func (p coreDNSProvider) groupEndpoints(changes *plan.Changes) map[string][]*end
 		grouped[ep.DNSName] = append(grouped[ep.DNSName], ep)
 	}
 	for i, ep := range changes.UpdateNew {
-		ep.Labels = changes.UpdateOld[i].Labels
 		log.Debugf("Updating labels (%s) with old labels(%s)", ep.Labels, changes.UpdateOld[i].Labels)
```

Comment:
```suggestion
		log.Debugf("Updating labels (%s) with old labels (%s)", ep.Labels, changes.UpdateOld[i].Labels)
```

probably needs a space, or remove space in first `labels(%s)`